.github/workflows/release.yml

```diff
@@ -1,15 +1,9 @@
 name: Release new version
 
 on:
-  workflow_dispatch:
-    inputs:
-      version:
-        description: "Release version (MAJOR.MINOR.PATCH, default: extracted from NEWS.md)"
-      preRelease:
-        description: "Is this a pre-release? (true/false, default: true if release version minor number is odd)"
-      dryRun:
-        description: "Dry Run (true/false, default: true)"
-        default: true
+  push:
+    tags:
+      - '*'
```

Comment:
Should it be limited to `1.*` tags to theoretically allow non-release tags in the repo?